Jamfile

```diff
@@ -272,6 +272,13 @@ rule building ( properties * )
 		result += <build>no ;
 	}
 
+	local VERSION = [ feature.get-values <cxxstd> : $(properties) ] ;
+	if ! $(VERSION) || $(VERSION) < 11
+	{
+		ECHO "libtorrent requires at least C++14. Specify <cxxstd>11 or higher" ;
```

Comment:
typo here `C++11`

Reply:
oops :). I'm not sure the boost version is new enough for this also. this is mostly an experiment